Libraries/LibLine/Editor.cpp

```diff
@@ -422,6 +422,69 @@ void Editor::handle_read_event()
     Utf8View input_view { StringView { m_incomplete_data.data(), valid_bytes } };
     size_t consumed_codepoints = 0;
 
+    enum Amount { kCharacter, kWord };
```

Comment:
uhhh...why not just `Character` and `Word`?

Reply:
Err, sorry. Fixed.